examples/cpp/advanced-filling.cpp

```diff
@@ -6,6 +6,7 @@
 #include <iomanip>
 #include <memory>
 #include <vector>
+#include <numeric>
```

Comment:
What do we need `numeric` for?

Reply:
Sorry, good catch! It was imported when I wanted to compute the shape of the subgrid as in here:
https://github.com/NNPDF/pineappl/blob/9c08d4f8c80f9ee797f6168adafef663b5ca70bc/examples/cpp/get-subgrids.cpp#L69-L70
which in the end I decided to hard-code but then forgot to remove.